recipe/meta.yaml

```
@@ -58,11 +43,12 @@ test:
     - mock
     - pytest-cov
   commands:
+    - export LC_ALL=en_US.UTF-8  # [osx]
     - python -c "import libarchive; libarchive.extract_file('test/hello_world.xar')"  # [not win]
     - python -c "import libarchive; libarchive.extract_file('test\\hello_world.xar')"  # [win]
     - python -c "import os, shutil, libarchive; shutil.copytree(os.path.dirname(libarchive.__file__), 'libarchive')"  # [unix]
-    - pytest --cov libarchive -vv  # [linux]
-    - pytest --cov libarchive -vv -k "not test_fd and not test_files and not test_buffers"  # [osx]
+    - pytest -vv --cov libarchive --cov-report term-missing:skip-covered --cov-fail-under=83 -k "not unicode_entries"  # [linux]
```

```suggestion
    - pytest -vv libarchive -k "not unicode_entries"  # [linux]
```
We don't need and should not run coverage. Coverage is only useful for upstream maintainers.